src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.scss

```diff
@@ -0,0 +1,35 @@
+@import "scss/constants";
+
+.Container {
+  &__room-preview {
+    position: absolute;
+  }
+
+  &__room-image {
+    width: 100%;
+    height: 100;
+    filter: none;
+    opacity: 1;
+    transition: filter 0.3s ease;
+
+    &--disabled {
+      filter: grayscale(100%);
+      opacity: 0.5;
+    }
+  }
+
+  &__room-title {
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    display: flex;
+    align-content: center;
+    justify-content: center;
+    background-color: opaque-black(0.5);
+    border-radius: $border-radius--xl;
+    padding: $spacing--xs $spacing--sm;
+    font-size: $font-size--sm;
+    transition: all 400ms cubic-bezier(0.23, 1, 0.32, 1);
```

Comment:
$transition-function here

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/e3b301bfa611c387d3e2ac24b7195994415ffc70